lib/domain/service/base/class_builder/freezed_class_builder.dart

```diff
@@ -31,31 +32,35 @@ class FreezedClassBuilder extends ClassBuilder {
     final classPartImport = '${_className.snakeCase}$importSuffix';
     final classFullName =
         '${_className.pascalCase}${_classNameSuffix.pascalCase}';
-    lines.add('import \'package:freezed_annotation/freezed_annotation.dart\';');
-    lines.addAll(_imports);
-    lines.addNewLine();
-    lines.add('part \'$classPartImport.freezed.dart\';');
-    lines.addNewLine();
-    lines.add('@freezed');
-    lines.add('class $classFullName with _\$$classFullName {');
+    lines
+      ..add("import 'package:freezed_annotation/freezed_annotation.dart';")
+      ..addAll(_imports)
+      ..addNewLine()
+      ..add("part '$classPartImport.freezed.dart';")
+      ..addNewLine()
+      ..add('@freezed')
+      ..add('class $classFullName with _\$$classFullName {');
     if (_baseConstructorProperties.isEmpty) {
-      lines.add('factory $classFullName() = _$classFullName;');
+      lines.add('const factory $classFullName() = _$classFullName;');
```

Comment:
@cozvtieg9, I mentioned above that I added const constructors, this const factory has been updated for this